common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/bootstrap/LwM2MTransportBootstrapServerConfiguration.java

```diff
@@ -0,0 +1,290 @@
+public class LwM2MTransportBootstrapServerConfiguration {
+    private PublicKey publicKey;
+    private PrivateKey privateKey;
+
+    @Autowired
+    private LwM2MTransportContextBootstrap contextBs;
+
+    @Autowired
+    private LwM2MTransportContextServer contextS;
+
+    @Autowired
+    private LwM2MBootstrapSecurityStore lwM2MBootstrapSecurityStore;
+
+    @Autowired
+    private LwM2MInMemoryBootstrapConfigStore lwM2MInMemoryBootstrapConfigStore;
+
+    @Primary
+    @Bean(name = "leshanBootstrapX509")
+    @ConditionalOnExpression("('${transport.lwm2m.bootstrap.secure.start_x509:false}'=='true')")
+    public LeshanBootstrapServer getLeshanBootstrapServerX509() {
+        log.info("Prepare and start BootstrapServerX509... PostConstruct");
+        return getLeshanBootstrapServer(this.contextBs.getCtxBootStrap().getBootstrapPortNoSecX509(), this.contextBs.getCtxBootStrap().getBootstrapSecurePortX509(), X509);
+    }
+
+    @Bean(name = "leshanBootstrapPsk")
+    @ConditionalOnExpression("('${transport.lwm2m.bootstrap.secure.start_psk:false}'=='true')")
+    public LeshanBootstrapServer getLeshanBootstrapServerPsk() {
+        log.info("Prepare and start BootstrapServerRsk... PostConstruct");
+        return getLeshanBootstrapServer(this.contextBs.getCtxBootStrap().getBootstrapPortNoSecPsk(), this.contextBs.getCtxBootStrap().getBootstrapSecurePortPsk(), PSK);
+    }
+
+    @Bean(name = "leshanBootstrapRpk")
+    @ConditionalOnExpression("('${transport.lwm2m.bootstrap.secure.start_rpk:false}'=='true')")
+    public LeshanBootstrapServer getLeshanBootstrapServerRpk() {
+        log.info("Prepare and start BootstrapServerRpk... PostConstruct");
+        return getLeshanBootstrapServer(this.contextBs.getCtxBootStrap().getBootstrapPortNoSecRpk(), this.contextBs.getCtxBootStrap().getBootstrapSecurePortRpk(), RPK);
+    }
+
+    public LeshanBootstrapServer getLeshanBootstrapServer(Integer bootstrapPortNoSec, Integer bootstrapSecurePort, LwM2MSecurityMode dtlsMode) {
+        LeshanBootstrapServerBuilder builder = new LeshanBootstrapServerBuilder();
+        builder.setLocalAddress(this.contextBs.getCtxBootStrap().getBootstrapHost(), bootstrapPortNoSec);
+        builder.setLocalSecureAddress(this.contextBs.getCtxBootStrap().getBootstrapSecureHost(), bootstrapSecurePort);
+
+        /** Create CoAP Config */
+        builder.setCoapConfig(getCoapConfig (bootstrapPortNoSec, bootstrapSecurePort));
+
+        /**  ConfigStore */
+        builder.setConfigStore(lwM2MInMemoryBootstrapConfigStore);
+
+        /** SecurityStore */
+        builder.setSecurityStore(lwM2MBootstrapSecurityStore);
+
+        /** Define model provider (Create Models )*/
+        builder.setModel(new StaticModel(contextS.getCtxServer().getModelsValue()));
+
+        /**  Create credentials */
+        LwM2MSetSecurityStoreBootstrap(builder, dtlsMode);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.